src/views/AskForHelp.jsx

```diff
@@ -24,6 +25,15 @@ export default function AskForHelp() {
 
   const isHotline = user && user.uid === hotlineId;
 
+  const askForHelpCollection = fb.store.collection('ask-for-help');
+
+  const [existingDocsForUser] = useCollectionData(
+    user ? askForHelpCollection.where('d.uid', '==', user.uid) : null,
+    { idField: 'id' },
+  );
+
+  const showAllowedRequestsHint = useMemo(() => !isHotline || (existingDocsForUser && existingDocsForUser.length > allowedOpenRequestsPerUser), [existingDocsForUser]);
```

Comment:
```suggestion
  const showAllowedRequestsHint = useMemo(() => !isHotline || (existingDocsForUser && existingDocsForUser.length > allowedOpenRequestsPerUser), [existingDocsForUser, isHotline]);
```

We have to add `isHotline` in the dependency array here.